src/ethereum_test_tools/common/types.py

```diff
@@ -524,7 +524,12 @@ def is_empty(self: "Account") -> bool:
             (self.nonce == 0 or self.nonce is None)
             and (self.balance == 0 or self.balance is None)
             and self.has_empty_code()
-            and (not self.storage or self.storage == {} or self.storage is None)
+            and (
+                not self.storage
+                or self.storage is None
+                or self.storage == {}
+                or not any(Storage.parse_key_value(v) != 0 for _, v in self.storage.items())
```

Comment:
How about?
```suggestion
                or all(v in [0, None] for v in self.storage.values())
```

Reply:
I like `any` a bit more because it aborts checking the rest of the items once any non-zero has been found, whereas `all` will always go through all the items. I'll commit the use of `.values()` though.

Reply:
Good point! :+1: